docs/Overview.rst

```diff
@@ -36,17 +36,67 @@ MVS Validation Plan
+   * - Electrical Grid + PV + Battery
+     - Reduced excess energy compared to Grid + PV scenario to charge the battery
+   * - Electrical Grid + Diesel Generator
+     - The diesel generator is only used if its LCOE is less than the grid price
+   * - Electrical Grid + Battery
+     - The grid is only used to feed the load
+   * - Electrical Grid + Battery + Peak Demand Pricing
+     - Battery is charged at times of peak demand and used when demand is larger
+   * - Electrical Grid (Price as Time Series) + Heat Pump + Heat Grid
+     - Heat pump is used when electricity_price/COP is less than the heat grid price
+     
+More tests can still be implemented with regard to the investment model within the MVS. Also, components with two input sources can also be tested.
+
+For sensitivity analysis, the behaviour of the MVS is studied by testing the effect of changing the value of the feed-in tariff for a fixed value of an asset's LCOE such that LCOE_ASSET is less than the electricity price. More input-output transformations can be investigated such as checking the randomness of supply between the electrical grid and a diesel generator when fuel_price/generator_efficiency is equal to electricity_price/transformer_efficiency. Another sensitivity analysis case could be used to know if a diesel generator actually replaces the consumption from the grid at times of peak demand--i.e., dispatch_price is less or equal to peak_demand_charge. The table below lists the implemented sensitivity analysis tests.
+
+.. list-table:: Sensitivity Analyses
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Sensitivity Analysis Test
+     - Expected Result
+   * - Comparing FIT to LCOE_ASSET
+     - Invest is maximum allowed capacity of asset for FIT values larger than LCOE_ASSET
+     
+Comparison to Other Models
+##########################
+
+So far, the MVS' results for a sector coupled system (electricity + hydrogen) are compared to those of HOMER for the same exact system. This comparison is important to highlight the similarities and differences between the two optimization models. On the electricity side, most of the values are comparable and in the same order of magnitude. The differences mainly show on the hydrogen part in terms of investment in an electrolyzer capacity (component linking the two sectors) and the values related to that. On another note, both models have different approaches for calculating the value of the levelized cost of a certain energy carrier and therefore the values are apart. 
+
+This validation method is very much in practice. However, one model cannot absolutely validate another model or claim that one is better than the other. This is why the focus should be on testing the correctness, appropriateness and accuracy of a model vis-à-vis its purpose. Since the MVS is an open source tool, it is important to use a validated model for comparison, but also similar open source tools like urbs and Calliope for instance.
```

Comment:
Please add a link to your thesis. 

"Same order of magnitude" is still quite far apart - are the values that far apart? Why?

What does "this validation method is very much in practice" mean?

You could again add a link to your thesis (add page number) or a paper, where possible models for comparison are listed.